dandi/validate_types.py

```diff
@@ -1,35 +1,53 @@
 from __future__ import annotations
 
 from dataclasses import dataclass
-from enum import Enum
+from enum import Enum, IntEnum
 from pathlib import Path
+from typing import Any
 
 
 @dataclass
 class ValidationOrigin:
-    name: str
-    version: str
-    bids_version: str | None = None
+    name: str  # Validator name
+    version: str  # Validator version
+    standard: str | None = (
+        None  # Standard being validated against # TODO: Enum for the standards??
+    )
+    standard_version: str | None = None  # Version of the standard
 
 
-class Severity(Enum):
+# TODO: decide on the naming consistency -- either prepend all with Validation or not
```

Comment:
@yarikoptic Are you talking about naming of the high level entities in this module, such as `Severity` and `ValidationOrigin`, or are you talking about the naming of the enum values within `Severity`?